index.html

```diff
@@ -24,6 +24,8 @@
   <link href="https://fonts.googleapis.com/css?family=Dancing+Script|PT+Sans" rel="stylesheet">
   <title>Days without Prince</title>
   <script src="src/googleanalytics.js"></script>
+  <script src="src/moment.js"></script>
+  <script src="src/moment-timezone-with-data.js"></script>
```

Comment:
Was these ones installed with npm?

If so, maybe they should be added in package.json? They will be added if you write:
npm install moment --save

I can of course add it myself after merging your pull request.